lib/counter_culture/collection.rb

```diff
@@ -0,0 +1,72 @@
+module CounterCulture
+  class Collection
+    include Enumerable
+
+    def initialize(arr=Array.new)
+      @arr = arr
+    end
+
+    def fix_counts(options={})
```

Comment:
It applies to a collection of counters. You want to perform an operation on all members of that collection. That is a valid collection method. This is a custom type specific collection. I'm not sure what you rules are about collections.